tests/test_build.py

```diff
@@ -59,6 +71,21 @@ def test_container_build_and_repo(container_per_test, host):
     assert sle_bci_repo.name == "SLE_BCI"
     assert sle_bci_repo.url == BCI_DEVEL_REPO
 
+    for repo_name in "SLE_BCI_debug", "SLE_BCI_source":
+        candidates = [repo for repo in repos if repo.name == repo_name]
+        assert len(candidates) in (0, 1)
+
+        if candidates:
+            container_per_test.connection.run_expect(
+                [0], f"zypper modifyrepo --enable {candidates[0].alias}"
```

Comment:
this mutates the state (enables the repository). why are we doing that? what does it test? we already tested earlier that it is listed. I think we should test that it is disabled by default perhaps?

Reply:
enabling the repository twice doesn't throw an error so this test doesn't do that..

Reply:
The last line in this test function runs `zypper ref`, so that would fail if the url were wrong.

Reply:
ah, so we're testing whether the repo is accessible. that makes sense. a comment would have been helpful.

Reply:
Done